aristamd/src/style/app.less

```diff
@@ -21,3 +21,99 @@
 @import "angulr/app.utilities.less";
 //
 // @import "angulr/app.butterbar.less";
+
+
+// Tabs CSS
+.wrapper-options{
```

Comment:
we need to check if some of the css added here can go in 1 or various file, app.less should have imports only